### PR TITLE
Fixes USB port names

### LINKED ISSUE #45: Provide documentation about usb devices ordering on raspi
The id assigned to each usb device should be deterministic.
Example:

 - USB0 -> microcontroller
 - USB1 -> rplidar

### PR DESCRIPTION
**Pull request by francocipollone**
# 🎉 New feature

Closes #45 

## Summary
 - Fixes USB port names to 
   - ttyUSB_ARDUINO: For the arduino device
   - ttyUSB_LIDAR: For the lidar scanner device.
 - Adds documentation.

## Checklist
- [x] Signed all commits for DCO
- [ ] Added tests
- [x] Added example and/or tutorial
- [x] Updated documentation (as needed)
- [ ] Updated migration guide (as needed)
- [ ] Consider updating Python bindings (if it affects the public API)

### CODE REVIEW

File: andino_hardware/README.md [hunk 1]
```diff
@@ -190,6 +190,91 @@ source install/setup.bash
 After this, you are good to go and use the robot!
```
Comment (jballoffet) [inline andino_hardware/README.md:190]: > After this, you are good to go and use the robot!

Is this still true or the steps below are mandatory to make it work?

If the steps below are indeed mandatory to make it work, I suggest changing the order here and placing it before the robot workspace creation steps.

**Review by jballoffet — CHANGES_REQUESTED** [review]
Left one comment, PTAL. Thanks!

File: andino_hardware/README.md (hunk 1 above)
Comment (francocipollone) [reply andino_hardware/README.md]: Nice catch. Done!

**Review by jballoffet — APPROVED** [review]
LGTM, thanks!

**Pull request merged**